kuma/static/js/search-suggestions.js

```diff
@@ -210,7 +210,13 @@
                         }
                     });
                 });
-                $searchInput.focus();
```

Comment:
I intentionally left it out since focusing the search box is a change from the current behavior.